ddmark/applyValidation.go

```diff
@@ -0,0 +1,180 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package ddmark
```

Comment:
We seem to follow a snake case convention for file names, could you rename this file to `apply_validation.go`?